policy/release/tasks_test.rego

```diff
@@ -510,7 +510,7 @@ _missing_tasks_warning(tasks) := {warning |
 	some task in tasks
 	warning := {
 		"code": "tasks.future_required_tasks_found",
-		"msg": sprintf("Task %q is missing and will be required in the future", [task]),
+		"msg": sprintf("Task %q is missing and will be required in the future on 2099-01-02T00:00:00Z", [task]),
```

Comment:
Just an idea, perhaps we can shorten it to "will be required on 2099-01-02T00:00:00Z".